tests/monitor/test_power.py

```
@@ -132,3 +135,142 @@ def test_cli_power_queries_and_integrates_same_domain(mocker: MockerFixture) ->
+        return index == 0
+
+
+class MockGPUs:
+    def __init__(self, count: int) -> None:
+        self._count = count
+
+    def __len__(self) -> int:
+        return self._count
+
+    def get_instant_power_usage(self, index: int) -> int:
+        raise ZeusGPUNotSupportedError("Unsupported")
```

Ruff (0.16.3)

[error] Undefined name `ZeusGPUNotSupportedError`

(F821)